packages/user-operation-controller/package.json

```diff
@@ -62,7 +62,7 @@
   "peerDependencies": {
     "@metamask/approval-controller": "^5.1.2",
     "@metamask/gas-fee-controller": "^13.0.0",
-    "@metamask/keyring-controller": "^12.2.0",
+    "@metamask/keyring-controller": "^13.0.0",
     "@metamask/network-controller": "^17.2.0",
     "@metamask/transaction-controller": "^23.0.0"
   },
```

Comment:
Only peer dependencies with major version change are updated due to https://github.com/MetaMask/core/pull/3881.